src/test/java/TestAmazonPageObject.java

```diff
@@ -0,0 +1,50 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class TestAmazonPageObject {
+
+    WebDriver driver;
+    @BeforeMethod
+    public void ouvrirChrome(){
+        driver = new ChromeDriver();
+        driver.get("https://www.amazon.fr/");
+
+        String acceptCookiesXpath = "sp-cc-accept";
+        WebElement cookiesButton = driver.findElement(By.id(acceptCookiesXpath));
+        cookiesButton.click();
+        driver.manage().window().maximize();
+
+    }
+    @AfterMethod
+    public void fermerChrome(){
+        driver.quit();
+    }
+
+
+    @Test
+    public void testAchatNintendoSwitch(){
+
+        //Arrange
+        String research = "Nintendo Switch";
+        String expectedResult = research;
```

Comment:
you could have done this:
`String expectedResult = research = "Nintendo Switch";`

or you can just use the same variable and don't declare it twice.